tools/circt-test/circt-test.cpp

```diff
@@ -86,6 +86,11 @@ struct Options {
   cl::opt<std::string> runner{
       "r", cl::desc("Program to run individual tests"), cl::value_desc("bin"),
       cl::init("circt-test-runner-sby.py"), cl::cat(cat)};
+
+  cl::opt<bool> runnerReadsMLIR{
+      "mlir-runner",
+      cl::desc("Pass the MLIR file to the runner instead of Verilog"),
+      cl::init(false), cl::cat(cat)};
```

Comment:
Maybe make this an enum "runnerInputFormat" because BTOR2 could also be an option?

Reply:
Good point. This option is currently just a hack to manually get the runners going. My idea was to have the runners themselves report what kind of input format they need, if the corresponding tool is installed on the system and available, and for which kinds of tests (formal, simulation, etc.) the runner is used for. This and the `-r` option will go away at that point, and will be replaced with a way for the user to pick a tool by name, or provide a list of tool preferences, e.g. `--tool sby,circt-bmc,vcf,jg`.